requirements.txt

```diff
@@ -44,3 +40,9 @@ ConcurrentLogHandler==0.9.1
 freezegun==0.3.9
 boto3==1.7.10
 xlrd==1.1.0
+numpy==1.9.0
+scikits.audiolab==0.11.0
+#apt-get install libyaml-dev (to allow the cloader)
```

Comment:
This makes sense to have in a separate INSTALL or README file instead of in requirements

Reply:
I agree, it is here for reference and because these should be installed before some of the python requirements. This will become clear when we move to Docker, and we currently have no up to date place where to put that information so I prefer to keep it like that for now. Nevertheless I added a small comment in the file for clarification 446b98f2536fc0d42d19b9d24bd6e611c2b41423.